reference/5.1/Microsoft.PowerShell.Utility/Remove-TypeData.md

```diff
@@ -3,111 +3,132 @@ external help file: Microsoft.PowerShell.Commands.Utility.dll-Help.xml
 keywords: powershell,cmdlet
 locale: en-us
 Module Name: Microsoft.PowerShell.Utility
-ms.date: 06/09/2017
+ms.date: 04/27/2020
 online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/remove-typedata?view=powershell-5.1&WT.mc_id=ps-gethelp
 schema: 2.0.0
 title: Remove-TypeData
 ---
 
 # Remove-TypeData
 
-## SYNOPSIS
+## Synopsis
 Deletes extended types from the current session.
 
-## SYNTAX
+## Syntax
 
 ### RemoveTypeDataSet (Default)
+
 ```
 Remove-TypeData -TypeData <TypeData> [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### RemoveTypeSet
+
 ```
 Remove-TypeData [-TypeName] <String> [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### RemoveFileSet
+
 ```
 Remove-TypeData -Path <String[]> [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Remove-TypeData** cmdlet deletes extended type data from the current session.
-This cmdlet affects only the current session and sessions that are created in the current session.
 
-You can add properties and methods to objects in Windows PowerShell by defining them in Update-TypeData commands and Types.ps1xml files.
-**Remove-TypeData** deletes those extended properties and methods from the current session.
-**Remove-TypeData** does not delete the Types.ps1xml files or delete any extended type definitions from the Types.ps1xml files.
-For more information about Types.ps1xml files, see about_Types.ps1xml (https://go.microsoft.com/fwlink/?LinkID=113274).
+The `Remove-TypeData` cmdlet deletes extended type data from the current session. This cmdlet
+affects only the current session and sessions that are created in the current session.
+
+You can add properties and methods to objects in PowerShell by defining them in `Update-TypeData`
+commands and `Types.ps1xml` files. `Remove-TypeData` deletes those extended properties and methods
+from the current session. `Remove-TypeData` does not delete the `Types.ps1xml` files or delete any
+extended type definitions from the `Types.ps1xml` files. For more information about Types.ps1xml
```

Comment:
The Types.ps1xml here got missed for codefence.

Reply:
fixed